app/controllers/calculators/take_whole_pot_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Calculators
+  class TakeWholePotController < CalculatorsController
+    def show
```

Comment:
Is it OK to override the [`show`](https://github.com/guidance-guarantee-programme/pension_guidance/blob/master/app/controllers/guides_controller.rb#L7) action defined in `GuidesController` that sets expiry cache headers?